### PR TITLE
feat: upgrade to Quince

### PR DESCRIPTION
**Pull request by jfavellar90**
This solves #22

### CODE REVIEW

File: tutorxqueue/templates/xqueue/build/xqueue/Dockerfile [hunk 1]
```diff
@@ -21,9 +21,9 @@ WORKDIR /openedx/xqueue
 RUN python -m venv /openedx/venv
 ENV PATH /openedx/venv/bin:${PATH}
 
-RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/openedx/.cache/pip,sharing=shared {% endif %}pip install --upgrade pip setuptools
+RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/openedx/.cache/pip,sharing=shared {% endif %}pip install setuptools==68.2.2 pip==23.2.1. wheel==0.41.2
```
Comment (regisb) [inline tutorxqueue/templates/xqueue/build/xqueue/Dockerfile]: Good catch. Could you please add links as comments to the pypi projects, just like in the other plugins?

Comment (jfavellar90) [reply tutorxqueue/templates/xqueue/build/xqueue/Dockerfile]: @regisb done

**Comment by jfavellar90** [comment]
@regisb since you simplified the nightly version management [here](https://github.com/overhangio/tutor-xqueue/commit/7317758ca93f7d192419e82ee5830a4b813fb41e), should I rebase this quince branch on top of the aforementioned changes?

**Comment by regisb** [comment]
Yes :)

**Pull request merged**